bedrock/settings/static_media.py

```diff
@@ -1464,7 +1436,7 @@
         ),
         'output_filename': 'js/growth-firstrun-test2-bundle.js',
     },
-    'home-2015': {
+    'home': {
         'source_filenames': (
             'js/base/mozilla-share-cta.js',
             'js/base/firefox-anniversary-video.js',
```

Comment:
Doesn't look like this video is on the home page anymore. Can probably remove this file from the bundle.